README.md

```diff
@@ -42,23 +43,23 @@ More about virtualenvs on https://docs.python.org/3/tutorial/venv.html
 Setup a virtual environment using:
 
 ```shell
-$ python3 -m venv .venv
+$ python3 -m venv venv
```

Comment:
oh, we are not keeping the folder hidden anymore.

Non blocking but for someone who does not understand these things well, they would get really confused of two "venv"s, about which one is command and which one is a folder.

Reply:
Oh! I completely misunderstood [your comment](https://github.com/dimagi/commcare-export/pull/235#discussion_r1488934391),

> Any reason to change that back to venv?
> I had made that change earlier to make the name more specific and avoid conflicts with any other setup since that is the name suggested by virtualenv documentation by default. We use the same name for CommCareHQ, `venv`.

I thought you were asking me to change `.venv` to `venv` in order to be consistent with its name in CommCare HQ. Sorry for my confusion.

> avoid conflicts

[virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/) puts all your virtual environments in one place, so it's important to give them unique names. But the Standard Library's [venv](https://docs.python.org/3/library/venv.html) is simpler, and doesn't have a special place for all your virtual environments. In my experience, most people just use a subdirectory in the repository's root named "venv", so there is no risk of conflicts. This convention is used in [GitHub's template `.gitignore` file for Python projects](https://github.com/github/gitignore/blob/main/Python.gitignore#L122-L129). I've just applied the same convention here.

Reply:
Okay.